client/src/app/modules/project-page/components/header.component.spec.ts

```diff
@@ -37,7 +27,7 @@ describe('ProjectHeaderComponent', () => {
         imports: [
           ProjectModule,
           LoadingIndicatorModule,
-          MatIconModule,
+          MatIconTestingModule,
```

Comment:
When I remember it correctly you would need both `MatIconModule` and `MatIconTestingModule` for mocking the icons.